.tsan_suppress

```diff
@@ -1 +1 @@
-race:tonic::response::Response*::into_inner
\ No newline at end of file
+race:core::ptr::drop_in_place
```

Comment:
Is a new line needed here?

Reply:
Fixed, but I'm hoping to investigate further and figure out a way to not need this.